pages/admin/user/[id].tsx

```diff
@@ -366,12 +374,26 @@ const UserProfile: React.FunctionComponent = () => {
               <p>{user && UserRoleLabel[user.defaultRole.type]}</p>
             </div>
           </div>
-          <h2 className="text-lg pb-5">Mentor Information</h2>
-          <div className="flex flex-row text-sm">
-            <p className="text-blue mr-20 w-24">Menteed Players</p>
-            <p>List</p>
-          </div>
-          <p className="text-lg font-semibold pb-10 pt-16">Account Changes</p>
+          {user?.defaultRole.type === "Mentor" && (
+            <div className="pb-16">
+              <h2 className="text-lg pb-5">Mentor Information</h2>
+              <div className="flex flex-row text-sm">
+                <p className="text-blue mr-20 w-24">Menteed Players</p>
+                {user?.roles
+                  ?.filter((role: DashboardRole) => role.relatedPlayer !== null)
+                  .map((role: DashboardRole) => {
+                    return (
+                      <Link href={`${role.relatedPlayer.id}`}>
```

Comment:
We actually want to link to the players profile (player dashboard). Since you already have the id you can link to "[role]/players/[id]"

Reply:
Also maybe look into changing the cursor so that it is a pointer when hovering over the linked player — I think it can be done with tailwind https://v1.tailwindcss.com/docs/cursor#app